source/layouts/application.html.haml

```diff
@@ -4,7 +4,6 @@
     %meta{charset: "utf-8"}
     %meta{content: "IE=edge,chrome=1", http: {equiv: "X-UA-Compatible"}}
     %meta{name: "viewport", content: "width=device-width, initial-scale=1"}
-    %link{:href => "https://fonts.googleapis.com/css?family=Playfair+Display", :rel => "stylesheet"}/
```

Comment:
I know you deleted this, but just a note for future reference, you could have used `stylesheet_link_tag` here.